script/Ciel/Day 10/10_01_CIEL10_7.txt

```diff
@@ -196,6 +196,7 @@ I have zero faith in the idea that I'm somehow reliable...
 [sha:bc9cf64fedd3acad751ba75f5b055d5cef3ca6a0]{
 -- Page 526, Offset 28553.
 -- 下級死徒の遺体が出てこないのは気になりますが、思い返してみればデパート地下の件がありました。ヴローヴが暴れた事で下級死徒の大部分は瓦礫に埋もれたのでしょう」
+// why aren't we using Lesser Dead Apostle for 下級死徒 as it is written in the glossary?
```

Comment:
Pretty sure this is actually referring to generally lower levels - actual Rank VI are independent, this is more about the hordes of ghouls or whatever he had.

Reply:
Reading back that makes sense